pulsar-io/kafka-connect-adaptor/src/main/resources/debezium-mysql-source-config.yaml

```diff
@@ -17,6 +17,12 @@
 # under the License.
 #
 
+##
+# NOTE: This could not be run, because it is lack of debezium dependency.
+# It is just an example of how to use this connector. 
+#
+# Debezium related code is moved to pulsar-io/debezium.
```

Comment:
why not just delete this file, since there is already one in `pulsar-io/debezium/mysql`?